ui/src/views/Documentation/Reference/Entry.jsx

```diff
@@ -109,8 +109,8 @@ export default class Entry extends Component {
       ? entry.args.concat(`{${entry.query.join(', ')}}`)
       : entry.args;
```

Comment:
I think this is actually the problem -- this puts the array `entry.args` in `parameters` by reference, and then pushes to it.  So every time this function runs, a new element is added to the end of `entry.args`.  We really shouldn't be modifying `entry.args`.  So I think the fix is to make a copy of the array here.  That can be done with `[...entry.args]`.

Reply:
Ah. yes. I completely missed this. Let me fix it right away